### PR TITLE
11.0 imp helpdesk mgmt 0

### PR DESCRIPTION
**Pull request by luffah**
Following the request : https://github.com/OCA/helpdesk/pull/42#issuecomment-514785080

On the portal, i found 2 functionals issue.
-  url:my/tickets view was raising an error on module name : s/helpdesk/helpdesk_mgmt/
-  error on ticket close because of a str type id

### CODE REVIEW

**Comment by luffah** [comment]
The view modification is not included is this PR because it was already in good shape.

File: helpdesk_mgmt/controllers/myaccount.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def portal_my_ticket(self, ticket_id=None, **kw):
         except AccessError:
             return request.redirect('/my')
         values = self._ticket_get_page_view_values(ticket_sudo, **kw)
-        return request.render("helpdesk.portal_helpdesk_ticket_page", values)
+        return request.render("helpdesk_mgmt.portal_helpdesk_ticket_page", values)
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt/controllers/myaccount.py]: helpdesk_mgmt/controllers/myaccount.py:111:80: E501 line too long (82 > 79 characters)

**Review by emagdalenaC2i — CHANGES_REQUESTED** [review]

File: helpdesk_mgmt/controllers/myaccount.py (hunk 1 above)
Comment (luffah) [reply helpdesk_mgmt/controllers/myaccount.py]: Added a commit for lint : https://github.com/OCA/helpdesk/pull/43/commits/630ee9e33dd46822f6ebfd49f057f1a0f5e32ab5

**Review by pedrobaeza — APPROVED** [review]
I would prefer to remove the `**kw` arg and directly put what we expect to receive, but OK for a bugfix

**Comment by pedrobaeza** [comment]
Please squash the flake8 fix to the commit that it fixes.

**Comment by luffah** [comment]
Squashed... As a mercurial user, I hope the change is ok.

**Comment by pedrobaeza** [comment]
Yes, thanks

**Review by emagdalenaC2i — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
@luffah Please, could you review #42 and then create another PR for v 12.0?

**Comment by luffah** [comment]
> @luffah Please, could you review #42 and then create another PR for v 12.0?

yes, i can (this afternoon or tomorrow).
To be sure not to be impolite: shall this PR on OCA include changes made by marcelsavegnago ? or shall i PR on marcelsavegnago repository ?

**Comment by pedrobaeza** [comment]
I would follow this order:

- Get #42 merged reviewing it for checking everything is OK.
- Do a new PR with this once merged.

**Pull request closed without merging**